view/static/testmap.html

```diff
@@ -102,8 +102,9 @@ <h2>DGGS Output</h3>
            <option value="11">11</option>
            <option value="12">12</option>
          </select>
+         <button id="queryDGGSGivenResolution">Go</button>
       </div>
-      <div id="loading" class="loading"><p>Loading</p></div>
+      <div id="loading" class="loading"><p>Loading... </p><p style="color: grey">Big areas and fine resolution will take a long time to process. </p></div>
```

Comment:
"Go" button works well and is a good addition :)

Can we add this message to appear if user selects resolution input of greater than 4 (to show up before user hits "Go"), instead of after which is the current behaviour? 

Also, can the message be: "DGGS Resolution 4 to 12 - (high resolution settings and big areas can take a much longer time)"
See issue https://github.com/CSIRO-enviro-informatics/AusPIX_DGGS_API/issues/42